Userland/Libraries/LibWeb/FileAPI/Blob.cpp

```diff
@@ -249,6 +252,64 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<Blob>> Blob::slice(Optional<i64> start, Opt
     return MUST_OR_THROW_OOM(heap().allocate<Blob>(realm(), realm(), move(byte_buffer), move(relative_content_type)));
 }
 
+// https://w3c.github.io/FileAPI/#stream-method-algo
+// https://w3c.github.io/FileAPI/#blob-get-stream
+WebIDL::ExceptionOr<JS::NonnullGCPtr<Streams::ReadableStream>> Blob::stream()
+{
+    auto& realm = this->realm();
+
+    // 1. Let stream be a new ReadableStream created in blob’s relevant Realm.
```

Comment:
Ideally, step 1 through 4 should be extracted to a private method called `Blob::get_stream()`, it will be reused by `Blob::text()` and `Blob::array_buffer()`.

Reply:
I was thinking about that originally, but figured Blob::text()` and `Blob::array_buffer()` could call `Blob::stream()` directly. I'll do that though, thanks, it is slightly closer to the spec text.